feature-about/src/main/java/io/github/droidkaigi/confsched2022/feature/about/ExternalServices.kt

```diff
@@ -0,0 +1,34 @@
+package io.github.droidkaigi.confsched2022.feature.about
+
+import androidx.annotation.DrawableRes
+
+sealed interface ExternalServices {
+    val iconRes: Int
+    val packageName: String
+    val url: String
+    val contentDescription: String
+
+    data class Twitter(
```

Comment:
Can we use object class or enum? I think we don't have to be able to be able to change this 👀

Reply:
@takahirom 
We have corrected it [here](https://github.com/DroidKaigi/conference-app-2022/pull/214/commits/41339549e0f1be9affe497371067fbd7ce27b52e).
It has already been approved, but please check again.🙇